src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -304,8 +307,24 @@ type FSharpCompilerServiceChecker(backgroundServiceEnabled, hasAnalyzers) =
     then ()
     else
       sdkRoot <- Some directory
-      sdkVersion <- Environment.latest3xSdkVersion directory
-      runtimeVersion <- Environment.latest3xRuntimeVersion directory
+      //TODO(CH): should this range be user-assignable somehow?
```

Comment:
similar point as in the background services: how should this be user-assignable?